tools/firtool/firtool.cpp

```diff
@@ -228,6 +228,17 @@ static cl::opt<bool>
                 cl::desc("run the reset inference pass on firrtl"),
                 cl::init(true), cl::cat(mainCategory));
 
+static cl::opt<bool> exportChiselInterface(
+    "export-chisel-interface",
+    cl::desc("generate a Scala Chisel interface to the top level "
+             "module of the firrtl circuit"),
+    cl::init(false), cl::cat(mainCategory));
+
+static cl::opt<std::string> chiselInterfaceOutDirectory(
```

Comment:
Directory being stdout ("-") seems a little strange, but I get this is a sentinel value of sorts to indicate it wasn't set and so should be emitted to stdout.

I'm guessing the directory option is not a filename so as to make it easier to dump interface files to a (re-used) location without needing to change this option to reflect a suitable filename?

Reply:
I agree `"-"` is a weird sentinel, I was using it since it is used elsewhere (see `inputFilename` and `outputFilename`). It's a directory since it makes sense to me implicitly name the generated file itself based on the circuit's top module name. But we can change this when we have a better idea of exactly what we want.

Reply:
It looks like other file options use an empty string as a sentinel which makes more sense to me, so I've updated the option to use that